telco-hub/configuration/reference-crs-kube-compare/optional/odf-internal/odfSubscription.yaml

```diff
@@ -4,7 +4,7 @@ kind: Subscription
 metadata:
   annotations:
     argocd.argoproj.io/sync-wave: "-40"
-  name: odf-operator
+  name: {{ .metadata.name }}
```

Comment:
This has a side-effect of matching this Subscription CR to all the sub-operator subscriptions that ODF creates when it is installed. Those then show various differences that aren't relevant since they are owned by ODF.